SpaceCommanderDummy.m

```diff
@@ -0,0 +1,7 @@
+#import <Foundation/Foundation.h>
```

Comment:
Let's do that

![hypnotoad](https://cloud.githubusercontent.com/assets/139364/9474664/0baf8e5e-4b30-11e5-9815-2b87f9fa9caa.gif)